bigtable/admin.go

```diff
@@ -1548,6 +1577,9 @@ type ClusterConfig struct {
 	// AutoscalingConfig configures the autoscaling properties on a cluster.
 	// One of NumNodes or AutoscalingConfig is required.
 	AutoscalingConfig *AutoscalingConfig
+
+	// Unit of scaling that allows more performance.
```

Comment:
Similar to rest of the comments, this comment should be
```suggestion
	// NodeScalingFactor specifies unit of scaling that allows more performance.
```

Reply:
Done